.eslintignore

```diff
@@ -3,3 +3,4 @@ dist
 es
 lib
 build
+**/fixtures
```

Comment:
why are we ignoring them in eslint?

Reply:
Coz the unit test files are partial component code and linter might complain for certain things. Also, `fixtures/expected` contains code that babel outputs, which is not linter compliant.

Reply:
ok